### DISCUSSION TITLE
How to convert Nextra to static HTML.

### DISCUSSION THREAD

**Discussion by hsiehyunju** [shuding/nextra#2512] [Q&A]
When I compile using 'pnpm build,' I see the following log, but I can't find any static files.

![Xnip2023-11-01_17-09-44](https://github.com/shuding/nextra/assets/25667798/f05129af-007c-4eee-85b2-0e492e11bc13)

I hope to be able to output static HTML and then deploy it to GitLab Pages. Is there any way to do it?

**Comment by dimaMachina** [comment]
use static exports https://nextjs.org/docs/pages/building-your-application/deploying/static-exports

**Reply by hsiehyunju** [reply]
Have you tried it? 
The solution you provided doesn't seem to work as described in the discussion linked below:
https://github.com/shuding/nextra/discussions/1853
